source/speech/commands.py

```diff
@@ -253,12 +260,27 @@ class CallbackCommand(BaseCallbackCommand):
 		otherwise it will block production of further speech and or other functionality in NVDA.
 	"""
 
-	def __init__(self, callback):
+	def __init__(self, callback: Union[SUPPORTED_CALLBACK_TYPES]):
```

Comment:
Is a class with a __call__ supported by this union?

Reply:
No. The only way to acoomplish this seems to be the use of collections.abc.Callable, but that also applies to classes itself. I'm reluctant to do this.